client/packages/requisitions/src/RequestRequisition/DetailView/IndicatorEdit/CustomerIndicatorInfo.tsx

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import {
+  Box,
+  ColumnDescription,
+  ColumnFormat,
+  createTableStore,
+  DataTable,
+  TableProvider,
+  TooltipTextCell,
+  useColumns,
+} from '@openmsupply-client/common';
+import { CustomerIndicatorInfoFragment } from '../../api';
+
+interface CustomerIndicatorInfoProps {
+  customerInfos?: CustomerIndicatorInfoFragment[] | null;
+}
+
+enum ColumnName {
+  Comment = 'Comment',
+  Value = 'Value',
+}
+
+const columnNameToLocal = (columnName: string) => {
+  switch (columnName) {
+    case ColumnName.Comment:
+      return 'label.comment';
+    case ColumnName.Value:
+      return 'label.value';
+    default:
+      return columnName;
+  }
+};
+
+const CustomerIndicatorInfo = ({
+  customerInfos,
+}: CustomerIndicatorInfoProps) => {
+  const columnNames = customerInfos?.[0]?.indicatorInformation?.map(
```

Comment:
Checking first element to get all the column names (should be the same for all elements after the first)